pep-9999.rst

```diff
@@ -85,23 +85,71 @@ PyPI disables new distribution uploads for ``bdist_egg`` distributions.
 Removal Process
 ===============
 
-This PEP proposes a removal process based on :pep:`527`, but with the following
-modifications:
+This PEP does **NOT** propose removing any existing files from PyPI, only
+disallowing new ones from being uploaded.
 
-* During the deprecation period, *all* packages will
-  `continue to be allowed <https://www.youtube.com/watch?v=XNyUALnj8V0>`_
-  to upload ``bdist_egg`` distributions, not just those that have previously
-  uploaded ``bdist_egg`` distributions.
-* During the deprecation period, uploading a ``bdist_egg`` distribution to PyPI
-  will trigger an email to all maintainers of the project, rather
-  than sending a single email at the beginning of the deprecation period.
+PyPI will provide a deprecation period of one month. At the beginning
+of the deprecation period, maintainers of projects that already have egg
+distributions will receive a one-time email informing them of the upcoming
```

Comment:
Oh, I was thinking this would be "uploaded one or more egg distributions *ever*", but maybe your edit is what @ewdurbin intended.

Either is fine by me, so punting to Ee for thoughts 🙂

Reply:
Yup, I inferred the above from @ewdurbin 's [comments above](https://github.com/python/peps/pull/3161#discussion_r1220779290):

> Volumes are low enough I'm actually game to do the one-time email with a one month deprecation.

which was followed by

> > all 1022 projects and the number of eggs they've uploaded, in 2023: https://gist.github.com/ewdurbin/d7939458f6205069adeeb18c454c9331

(And the query in the linked Gist confirms that the list of projects are indeed those uploading an egg in 2023).

From this, I inferred that given that the referenced "volumes" and list of projects are those for 2023 (which would be at least ≈six months prior to the start of the deprecation period when this PEP is likely implemented, and following the release of Buildout versions supporting wheels on all active branches, and the EoL of those that support eggs), Ee was likely referring to this being the list of projects that would be notified, but I could definitely be interpreting that wrong. We'll see what they say.

Reply:
Notify anyone who has uploaded in an egg in 2023.

Reply:
Thanks @ewdurbin !